packages/ckeditor5-ui/src/editorui/editorui.ts

```diff
@@ -299,6 +305,25 @@ export default abstract class EditorUI extends /* #__PURE__ */ ObservableMixin()
 		this._focusableToolbarDefinitions.push( { toolbarView, options } );
 	}
 
+	/**
+	 * Stores default location config for given menu bar item.
+	 */
+	public addMenuBarItem(
```

Comment:
I know this function is so simple, but could we add one unit tests for future protection?